docs/en/rs-school-trainer.md

```diff
@@ -1,36 +1,45 @@
-•	Offline time should include 1-2 breaks
-•	Remove all unnecessary information from presentations. All the main links that can be  useful for further reading on the topic should be given at the end of the presentation
-•	Before giving the presentation, omit all materials / slides that are out of date. For example, nobody is interested in layout features for IE6 or handling memory leaks in IE8
-•	The turned on webcam at webinars makes the lecture more cordial
-•	During offline classes, communicate with the entire audience, and not with the front rows
-•	It is recommended to use contrasting colors in the presentation, as all lectures are shown to the audience on the projector
-•	For specific and rarely used things, it is better to leave a link to the material, do not describe them in detail
-•	Make more simple slides rather than one huge
-•	To present an API analysis of any functionality, it is better to organize workshops, not just enumerate them
+
+- No more than 2 hours of theory for each lesson. Theoretical knowledge should be covered by practice - task, test or other activity. Everything that is put into practice is quickly forgotten.
+- Offline time should include 1-2 breaks
+- Remove all unnecessary information from presentations. All the main links that can be  useful for further reading on the topic should be given at the end of the presentation
+- Before giving the presentation, omit all materials / slides that are out of date. For example, nobody is interested in layout features for IE6 or handling memory leaks in IE8
+- The turned on webcam at webinars makes the lecture more cordial
+- During offline classes, communicate with the entire audience, and not with the front rows
+- It is recommended to use contrasting colors in the presentation, as all lectures are shown to the audience on the projector
+- For specific and rarely used things, it is better to leave a link to the material, do not describe them in detail
+- Make more simple slides rather than one huge
+- To present an API analysis of any functionality, it is better to organize workshops, not just enumerate them
 
 ### Organizing offline classes
-We are constantly looking for those who want to conduct face-to-face lectures, student meetings, beerjs, office tours, etc. At the face-to-face event, you can give a lecture from the RS School curriculum - https://github.com/rolling-scopes-school/lectures or just talk about working on a project, passing interviews, etc.
+
+We are constantly looking for those who want to conduct face-to-face lectures, student meetings, beerjs, office tours, etc. At the face-to-face event, you can give a lecture from the RS School curriculum - [https://github.com/rolling-scopes-school/lectures](https://github.com/rolling-scopes-school/lectures) or just talk about working on a project, passing interviews, etc.
```

Comment:
We are constantly looking for those who want to conduct face-to-face lectures, student meetups, beerjs, office tours, etc. During face-to-face event, you can give a lecture from the RS School curriculum - [https://github.com/rolling-scopes-school/lectures](https://github.com/rolling-scopes-school/lectures) or just talk about working on a project, passing interviews, etc.